### ISSUE TITLE
Questions about publishing on Pypi

### ISSUE THREAD

**Issue by Lucas-C** [nicovillanueva/marathon-deploy#2]
Hi !

Thank you for open-sourcing those scripts !
We used them over the last year at work, and they were very useful.

We made some slight improvements to some of them, and just thought today that we could publish those on GitHub.

I've just created a public fork https://github.com/voyages-sncf-technologies/marathon-deployer
And we will soon upload additional commits to it,
and submit a PR if you wish to incorporate them.

We currently upload it as a package on our internal Python repository,
and we thought it could also be published on the public Pypi repo.

Hence I have some questions:
- do you plan to publish it on Pypi ?
- else, would you agree that we do so ?
- if so, do you have a prefered license ? Maybe "Apache 2" ?

Kind regards @nicovillanueva

**Comment by nicovillanueva** [comment]
Bonjour !

What an honor to learn that you've been using them!
Yes, please do send a PR with your changes. As you probably noticed, I published the project and basically forgot about it, but I'm now installing `miniDCOS` to give it a bit of love and maintenance.
Once your PR is merged and whatever changes I come up with are done, I'll put it up on PyPi, of course.

Now, given that you've been using it, do you have any features you'd like to see implemented? And, any suggestions for a catchier name than just "deployer"?

Merci beaucoup, @Lucas-C !

**Comment by Lucas-C** [comment]
Here is our PR: https://github.com/nicovillanueva/marathon-deployer/pull/3

It includes some configuration for [Travis CI](https://travis-ci.org) (are you familiar with it ?) and a suggestion of steps to upload a package on Pypi: https://github.com/voyages-sncf-technologies/marathon-deployer/tree/version-3#pypi-publishing (wich could be done with Travis CI too if you wish)

No hurry, but could you please take the time to review our changes and give us some feedback ?

**Comment by nicovillanueva** [comment]
Yeah, the changes look great. I'll merge them right now.
I know of Travis CI, but never really used it, so this definitely is a good chance to give it a go.

I'll leave this ticket open until this is up on PyPi. Thank you so, so much for your help.

**Comment by nicovillanueva** [comment]
@Lucas-C , the project is up in PyPi: https://pypi.org/project/marathon-deploy/3.0.2/
It's been a while since I published anything there, so the working version is 3.0.2. The previous ones are flukes.

Notice that there are a two potentially breaking changes: I changed the repository name to "marathon-deploy" to be consistent with the executable's naming; and I changed the executables from `*-marathon` to `marathon-*` (e.g., the executable is now `marathon-deploy`).

I'll keep working on the project so feel free to raise new issues with either bugs or feature requests. Also, PRs are more than welcome.
